### DISCUSSION TITLE
NFC library issue with nrf52840 sense

### DISCUSSION THREAD

**Discussion by KylianBOIDARD** [Seeed-Studio/wiki-documents#111] [Q&A]
Hello everyone,
I was trying to use NFC on nrf52840 sense as explained in the wiki example.
I am getting the error : "NFC.h : No such file or directory" for the normal board, 
and the error "NFC was not declared in this scope" for the mbed board. 
This seems to be a library issue but i have no idea how to fix it. 
Your help would be very appreciated, thanks in advance.

**Comment by MatthewJeffson** [comment]
Hi, the NFC library is currently under updating and will be published soon.
